app/views/helpers/injected/LanguageSelection.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import cats.syntax.eq._
+@import cats.instances.string._
+
+@import config.FrontendAppConfig
+
+@this()
+@()(implicit messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
`appConfig` is not used, please remove it.
```suggestion

@this()
@()(implicit messages: Messages)
```